sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -4,6 +4,7 @@ standalone-mode = false
 authentication-mode = "aws"
 server-tls-bootstrap = true
 cloud-provider = "aws"
+pod-infra-container-image = "localhost/amazon/pause-bottlerocket:0.1.0"
```

Comment:
We can just drop this in addition to the settings generator going away.

And then either:
* set `POD_INFRA_CONTAINER_IMAGE` in the `kubelet-env` template to the hardcoded value
* or else set the hardcoded value in `kubelet-exec-start-conf`

Reply:
Oh, we'd also need to update these two files with the hardcoded value:
```
packages/containerd/containerd-config-toml_k8s_containerd_sock
packages/containerd/containerd-config-toml_k8s_nvidia_containerd_sock
```

Reply:
I thought about just always using the built in one, but thought there might be some weird use case where someone runs a specific pod infra container image.  I'm not sure what that would be, WDYT?

Reply:
> I thought about just always using the built in one, but thought there might be some weird use case where someone runs a specific pod infra container image. I'm not sure what that would be, WDYT?

I haven't heard of a use case like that. Only the obvious one, where someone might want to override the image so it's pulled from a different registry in a restricted or air-gapped environment. Building it solves that problem nicely!